packages/front-end/enterprise/components/Dashboards/DashboardEditor/DashboardUpdateDisplay.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useContext, useMemo } from "react";
+import { Flex, Text } from "@radix-ui/themes";
+import { ago } from "shared/dates";
+import { PiArrowClockwise, PiInfo, PiLightning } from "react-icons/pi";
+import clsx from "clsx";
+import { dashboardCanAutoUpdate } from "shared/enterprise";
+import {
+  DashboardBlockInterfaceOrData,
+  DashboardBlockInterface,
+} from "back-end/src/enterprise/validators/dashboard-block";
+import LoadingSpinner from "@/components/LoadingSpinner";
+import Tooltip from "@/components/Tooltip/Tooltip";
+import Button from "@/components/Radix/Button";
+import { useUser } from "@/services/UserContext";
+import MoreMenu from "@/components/Dropdown/MoreMenu";
+import ViewAsyncQueriesButton from "@/components/Queries/ViewAsyncQueriesButton";
+import { DashboardSnapshotContext } from "../DashboardSnapshotProvider";
+
+function SnapshotStatusSummary({
+  blocks,
+  enableAutoUpdates,
+}: {
+  blocks: DashboardBlockInterfaceOrData<DashboardBlockInterface>[];
+  enableAutoUpdates: boolean;
+}) {
+  const {
+    settings: { updateSchedule },
+  } = useUser();
+  const {
+    experiment,
+    defaultSnapshot: snapshot,
+    refreshError,
+    allQueries,
+  } = useContext(DashboardSnapshotContext);
+  const numFailed = useMemo(
+    () => allQueries.filter((q) => q.status === "failed").length,
+    [allQueries]
+  );
+
+  if (!snapshot) return null;
+
+  const autoUpdateEnabled =
+    enableAutoUpdates &&
+    dashboardCanAutoUpdate({ blocks }) &&
+    updateSchedule?.type !== "never" &&
+    experiment?.autoSnapshots;
+  const timeTillUpdate = experiment?.nextSnapshotAttempt;
+
+  const textColor = refreshError || numFailed > 0 ? "red" : undefined;
+  const content = refreshError
+    ? "Update Failed"
+    : numFailed > 0
+    ? "One or more queries failed"
+    : snapshot.runStarted
+    ? `Updated ${ago(snapshot.runStarted).replace("about ", "")}`
+    : "Not started yet";
+  const tooltipBody = refreshError ? refreshError : undefined;
```

Comment:
Is this true for stopped experiments (do the dashboards keep auto-updating)?
<img width="1357" height="526" alt="image" src="https://github.com/user-attachments/assets/3bdd41dd-a4e1-41a1-af84-3769e07d0f2e" />